src/main/java/core/basesyntax/AreaOfTheFigures.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaOfTheFigures {
```

Comment:
```suggestion
public interface AreaCalculator{
```

про названия интерфейсов есть в чеклисте https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced